x/gov/migrations/v5/store.go

```diff
@@ -36,7 +36,7 @@ func MigrateStore(ctx sdk.Context, storeService corestoretypes.KVStoreService, c
 	}
 
 	defaultParams := govv1.DefaultParams()
-	params.ExpeditedMinDeposit = defaultParams.ExpeditedMinDeposit
+	params.ExpeditedMinDeposit = params.MinDeposit // Use regular `min_deposit` as `expedited_min_deposit`
 	params.ExpeditedVotingPeriod = defaultParams.ExpeditedVotingPeriod
```

Comment:
blocking comment: 
1. already discussed this but we should specify `min_deposit_ratio` too. 
2. let's triple check that we didn't miss any new params

Reply:
1. Added a comment for `min_deposit_ratio` noting that this is new parameter and we are using the native default `0.01`